test/config-luatest/helpers.lua

```diff
@@ -185,31 +197,43 @@ end
 -- Start a server, write a new script/config, reload, run a
 -- verification function.
 --
+-- * opts.roles
 -- * opts.script
 -- * opts.options
 -- * opts.verify
 --
 --   Same as in success_case().
 --
+-- * opts.roles_2
+--
+--   A new list of roles to prepare before config:reload().
+--
 -- * opts.script_2
 --
 --   A new script to write into the main.lua file before
 --   config:reload().
 --
+-- * opts.options_2
+--
+--   A new options for config:reload().
+--
 -- * opts.verify_2
 --
 --   Verify test invariants after config:reload().
 local function reload_success_case(g, opts)
+    assert(opts.options)
+    local roles_2 = opts.roles_2
     local script_2 = opts.script_2
-    local options = assert(opts.options)
+    local options_2 = opts.options_2
```

Comment:
It is also added in #9088. I guess that I'll push it first. So, this comment is a reminder to rebase afterwards.

Reply:
PR #9088 lands to `master`.

Reply:
Rebased.